targets/TARGET_Silicon_Labs/TARGET_EFM32/emlib/Changes_emlib.txt

```diff
@@ -46,17 +46,6 @@ Gecko Platform 2.6.0.0 (32-bit MCU SDK 5.8.0.0)
  - Added LETIMER_CounterGet() and LETIMER_CounterSet() functions to
    em_letimer.
  - Added TIMER_SyncWait() function to em_timer.
- * Changed
```

Comment:
Was there a reason for removing this section?

Reply:
It was a duplicate of the "Added" section.

Reply:
Whoops. Thanks for the keen eye, I'll notify the internal team.